apps/sms/locales/sms.en-US.properties

```diff
@@ -315,6 +315,10 @@ contact-photo.ariaLabel = Contact photo
 unread-thread.ariaLabel = Unread
 has-draft.ariaLabel = Has draft
 is-draft.ariaLabel = Draft
+optionsButton.ariaLabel = Options
+settingsButton.ariaLabel = Settings
+composeLink.ariaLabel = Compose
+callButton.ariaLabel = Call
```

Comment:
@julienw : 
Generally we are trying to be as terse as possible since there's a lot of verbal content that the screen reader user needs to comprehend, thus we are trying to be short as possible. Also there's a lot of contextual information that the user has since they know where they are located within the UI. If what I have is too long how about this:
* options => Show actions
* I would leave settings as is because it's a settings button and having settings panel button is a little confusing
* compose => Compose message
* call => Call contact
What do you think?